main.tf

```diff
@@ -68,13 +68,27 @@ resource "google_compute_router" "this" {
   ]
 }
 
+resource "google_compute_address" "address_nat" {
```

Comment:
Following Terraform's best practices, you shouldn't repeat "address" word in this resource's name